### PR TITLE
Rename KAKAROT_CONTRACT_ACCOUNT_CLASS_HASH to ACCOUNT_CLASS_HASH

### PR DESCRIPTION
**Pull request by dubzn**
The constant called KAKAROT_CONTRACT_ACCOUNT_CLASS_HASH will be renamed to ACCOUNT_CLASS_HASH.
Closes [#124](https://github.com/sayajin-labs/kakarot-rpc/issues/124)
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

Issue Number:  [#124](https://github.com/sayajin-labs/kakarot-rpc/issues/124)

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

File: core/src/client/mod.rs [hunk 1]
```diff
@@ -1005,7 +1004,7 @@ impl KakarotClient for KakarotClientImpl {
             }
         }
 
-        let kakarot_class_hash = FieldElement::from_hex_be(KAKAROT_CONTRACT_ACCOUNT_CLASS_HASH)
+        let kakarot_class_hash = FieldElement::from_hex_be(ACCOUNT_CLASS_HASH)
```
Comment (dubzn) [inline core/src/client/mod.rs:1007]: How do you see changing the name of the variable **kakarot_class_hash** to **account_class_hash**?

File: core/src/client/mod.rs [hunk 2]
```diff
@@ -1005,7 +1004,7 @@ impl KakarotClient for KakarotClientImpl {
...
             .map_err(|e| {
                 KakarotClientError::OtherError(anyhow::anyhow!(
                     "Kakarot Failed to convert Kakarot custom proxy contract class hash to FieldElement: {}",
```
Comment (dubzn) [inline core/src/client/mod.rs:1010]: Change the message to _"Kakarot Failed to convert account contract class hash to FieldElement: {}"_?

Comment (Eikix) [reply core/src/client/mod.rs:1010]: Good idea

File: core/src/client/mod.rs (hunk 1 above)
Comment (Eikix) [reply core/src/client/mod.rs:1007]: I think @ClementWalter already made this change in #125

Comment (dubzn) [reply core/src/client/mod.rs:1007]: He actually [remove the constants](https://github.com/sayajin-labs/kakarot-rpc/pull/125/files#diff-3f796ff7496f37e2f6e670ee3eea1fe7a4fb28955ab6bb1867b60037f593a0f6L4), maybe this PR should be closed :thinking:

Comment (Eikix) [reply core/src/client/mod.rs:1007]: Good point, you're right

**Comment by Eikix** [comment]
Closing this PR as suggested, as #125 already resolves the purpose of this PR.
Thank you for your pointing it out:)

**Pull request closed without merging**